deployment/v3/mosip/regclient/install.sh

```diff
@@ -55,7 +31,6 @@ function installing_regclient() {
     --set regclient.healthCheckUrl=$HEALTH_URL \
     --set regclient.hostName=$INTERNAL_HOST \
     --set istio.host=$REGCLIENT_HOST \
-    --wait \
```

Comment:
Add `--wait --wait-for-jobs`

Reply:
added wait
there no wait command separatly for pre-install-jobs....wait is enough
the --wait flag will ensure that Helm waits for the completion of the pre-install job as well. This means:

The Helm install process will start.
The pre-install job will run.
Helm will wait for the pre-install job to complete.
After the pre-install job completes, Helm will continue with the rest of the chart installation.
Helm will then wait for all the other resources to be in a ready state.